### PR TITLE
[*] BREAKING feat(css): styles are bundled within JS

### PR DESCRIPTION
**Pull request by Barackos**

### CODE REVIEW

File: examples/main/shared/theme/text-hashtag.theme.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .hashtag {
-  color: red;
+  color: red !important;
```
Comment (Barackos) [inline examples/main/shared/theme/text-hashtag.theme.scss:2]: Theme was overridden by `hashtag.css`.
I think it's an issue worth considering. Maybe other classnames are overridden & not covered in test-env.

File: packages/wrapper/web/src/themeStrategy/themes/editor-common.ts [hunk 2]
```diff
@@ -80,7 +81,9 @@ export default function editorCommon(colors: PaletteColors) {
         backgroundColor: actionColor,
       },
     },
-    checkbox_icon: {},
+    checkbox_icon: {
+      color: utils.fallbackColorBright,
+    },
```
Comment (Barackos) [inline packages/wrapper/web/src/themeStrategy/themes/editor-common.ts:86]: damages made by the transition & fixed here.

File: packages/wrapper/web/tsconfig.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
     "importHelpers": true,
     "target": "ES5",
     "jsx": "react",
-    "lib": ["dom", "ESNext"],
+    "lib": ["dom", "ESNext", "ES5"], //ES5 is for test-env, do not remove.
```
Comment (Barackos) [inline packages/wrapper/web/tsconfig.json:16]: for some reason, wrapper test-env cannot work without it

Comment (mharsat) [reply packages/wrapper/web/tsconfig.json:16]: Are you sure? The test env runs successfully in my branch without ES5. moreover the ES5 lib is s subset of the ESNext lib

File: examples/main/shared/theme/text-hashtag.theme.scss (hunk 1 above)
Comment (lxgreen) [reply examples/main/shared/theme/text-hashtag.theme.scss:2]: maybe the specificity once again, or the css ordering in the document

File: packages/wrapper/web/src/themeStrategy/themes/common.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ export default function commonStyles(colors: PaletteColors) {
   return {
     editor: {
       background: bgColor,
-      color: textColor,
+      color: `${textColor} !important`,
```
Comment (lxgreen) [inline packages/wrapper/web/src/themeStrategy/themes/common.ts:7]: ?

File: packages/wrapper/web/src/themeStrategy/themes/editor.ts [hunk 5]
```diff
@@ -14,7 +14,7 @@ export default function editor(colors: PaletteColors, utils: ThemeUtils) {
     },
     footerToolbar: {
       '&$staticToolbar': {
-        background: bgColor,
+        background: `${bgColor} !important`,
```
Comment (lxgreen) [inline packages/wrapper/web/src/themeStrategy/themes/editor.ts:17]: ??

**Comment by Barackos** [comment]
Closing for now, till we further research the best practice

**Pull request closed without merging**